bcdaworker/worker/worker.go

```diff
@@ -0,0 +1,387 @@
+package worker
+
+import (
+	"bufio"
+	"context"
+	"database/sql"
+	"encoding/json"
+	goerrors "errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strconv"
+
+	"github.com/CMSgov/bcda-app/bcda/client"
+	"github.com/CMSgov/bcda-app/bcda/models"
+	fhirmodels "github.com/CMSgov/bcda-app/bcda/models/fhir"
+	"github.com/CMSgov/bcda-app/bcda/responseutils"
+	"github.com/CMSgov/bcda-app/bcda/utils"
+	"github.com/CMSgov/bcda-app/bcdaworker/repository"
+	"github.com/CMSgov/bcda-app/bcdaworker/repository/postgres"
+	"github.com/CMSgov/bcda-app/conf"
+	newrelic "github.com/newrelic/go-agent"
+	"github.com/pborman/uuid"
+	"github.com/pkg/errors"
```

Comment:
Now that I'm working through this a bit, could with alias this package to something like `pkgerrors` and leave the go std `errors`?

Reply:
ah that's a great idea

Reply:
Will create a ticket to address this in all packages.